tests/utils/atomWithObservable.test.tsx

```diff
@@ -41,7 +41,7 @@ it('writable count state', async () => {
     // is this usual to delay the subscription?
     setTimeout(() => {
       observable.subscribe(subject)
-    }, 500)
+    }, 100)
```

Comment:
This test is still unstable. This might not affect anything, but just hoping.

Reply:
I analysed the issue. The problem is, that the `onMount` subscription is created __after__ the dispatch of the button. My logs (I hope it makes sense):

![image](https://user-images.githubusercontent.com/9933601/168534761-ad3662e7-ee85-4ccf-b6d0-d52cbf1ea6aa.png)

I can look into this issue this evening.

Reply:
Thanks for taking a look. Please work on a separate PR as I will merge this some time eventually.

btw, contributions on `atomWithObservable` are very welcome, because currently no contributors on it are active except for you. Please keep it going.

Reply:
Sounds good to me!